ForumProject/notifications/models.py

```diff
@@ -49,20 +44,99 @@ def __str__(self):
         return f'Notification of {self.date_time} to {addressee} on Project {project_name}'
 
 
-
     class Meta:
         # Add model-level constraint
         constraints = [
             models.CheckConstraint(check=models.Q(project__isnull=False) | models.Q(investor__isnull=False), name='project_or_investor_required'),
         ]
 
-class NotificationPreferences(models.Model):
-    startup = models.OneToOneField(Startup, 
+
+class StartupNotificationPrefs(models.Model):
+    startup = models.OneToOneField(
+        Startup, 
         on_delete=models.CASCADE,
-        related_name='notice_preferences',
+        related_name='startup_notice_prefs',
         null=True
     )
-    email_on_followers_change = models.BooleanField(default=True, verbose_name='Followers change email notices')
-    email_on_share_subscription = models.BooleanField(default=True, verbose_name='Subscriptions change email notices')
-    in_app_on_followers_change = models.BooleanField(default=True, verbose_name='Followers change in_app notices')
-    in_app_on_share_subscription = models.BooleanField(default=True, verbose_name='Subscriptions change in_app notices')
+    # preferences for Notifications about Investors' actions on PROJECTS
+    email_project_on_investor_interest_change = models.BooleanField(default=True, verbose_name='Email Investor-Project interest')
+    push_project_on_investor_interest_change = models.BooleanField(default=True, verbose_name='Push Investor-Project interest')
+    # preferences for Notifications about Investors' actions on STARTUPS
+    email_startup_on_investor_interest_change = models.BooleanField(default=True, verbose_name='Email Investor-Startup interest')
+    push_startup_on_investor_interest_change = models.BooleanField(default=True, verbose_name='Push Investor-Startup interest')
+
+    active_email_preferences = models.CharField(
+        max_length=100, 
+        blank=True, 
+        default='Project follower list or subscription share change, Startup sibscribers list changed'
+    )
+    active_push_preferences = models.CharField(
+        max_length=100, 
+        blank=True, 
+        default='Project follower list or subscription share change, Startup sibscribers list changed'
```

Comment:
It might be typos in default values ('sibscribers' to 'subscribers').

Reply:
Thank you! The typo has been corrected.